nengo_viz/static/viz_slider.js

```diff
@@ -151,6 +127,41 @@ VIZ.Slider = function(args) {
 VIZ.Slider.prototype = Object.create(VIZ.Component.prototype);
 VIZ.Slider.prototype.constructor = VIZ.Slider;
 
+VIZ.Slider.prototype.set_value = function(graph, slider, min_in, max_in, value) {
+    /** check if click was the middle mouse button */
```

Comment:
Actually, wait a second, no, ignore all of that.  You already have a min_value and max_value as part of the ```slider.scale``` object.  Just use that, and all this extra code goes away!